ui/package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
     "format": "npx prettier src --write",
-    "test": "NODE_ENV=development jest",
```

Comment:
This was something we unnecessarily added when implementing the dummy axios data, to support testing in vercel. It actually is not needed, and is resulting in a lot of unnecessary errors to be logged when running unit tests.